src/main/java/io/quarkus/registry/app/model/ExtensionRelease.java

```diff
@@ -53,7 +61,7 @@ public class ExtensionRelease extends BaseEntity {
      * The version above formatted for max and order-by operations
      */
     @Column(updatable = false, length = 100)
-    private String versionSortable;
+    public String versionSortable;
```

Comment:
Why is this public?

Reply:
Good spot - I had to make it public when DatabaseRegistryClient was doing the sorting, but I can put it back now.